src/components/AddPartnerModal/AddPartnerModal.js

```diff
@@ -0,0 +1,242 @@
+import React from 'react';
+import {
+    Button,
+    Col,
+    Form,
+    FormGroup,
+    Input,
+    Label,
+    Modal,
+    ModalBody,
+    ModalFooter,
+    ModalHeader,
+    Row,
+  } from 'reactstrap';
+  import stateRequests from '../../helpers/data/stateRequests';
+import utils from '../../../src/helpers/utils/utility';
+
+const defaultPartner = {
+    name: '',
+    description: '',
+    street: '',
+    city: '',
+    state: '',
+    zipcode: '',
+};
+
+class AddPartnerModal extends React.Component {
+    state = {
+        modal: false, 
+        backdrop: 'static', 
+        newPartner: defaultPartner,
+        usStates: [],
+        descriptionMaxLength: 250,
+        descriptionCharCount: 250,
+    };
+
+    toggle() {
+        this.setState({
+            modal: !this.state,
+        });
+    }
+
+    modalClosed() {
+        const { modalCloseEvent } = this.props;
```

Comment:
The form does not seem to be reset when the Modal closes after selecting Edit on a partner and then closing the modal.  If you click Add after doing this the new Partner Modal has the Partner info of the one you clicked Edit on.